Makefile

```diff
@@ -14,7 +14,7 @@ define docker_run_flags
 --rm \
 -v /var/run/docker.sock:/var/run/docker.sock \
 --env-file $(PWD)/etc/env.conf \
--P \
+-p 8081:8080 \
```

Comment:
Can you let the random port directive? I usually run this container along side a service discovery container.